src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 public class WorkWithFile {
+
+    private static final Logger LOGGER = Logger.getLogger(WorkWithFile.class.getName());
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
this method should contain three calls of other methods: readFile() - returns content, generateReport() -(you pass contents here), writeToFile()-(you pass report and filename here ), this will be a good code decomposition. example:
```
        String data = readFile(fromFileName);
        String report = generateReport(data);
        writeToFile(toFileName, report);
```

All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)